bundle/manifests/sonataflow-operator-controllers-config_v1_configmap.yaml

```diff
@@ -58,6 +58,9 @@ data:
       - groupId: org.kie
         artifactId: kie-addons-quarkus-persistence-jdbc
         version: 999-20240912-SNAPSHOT
+    # If true, the workflow deployments will be configured to send accumulated workflow status change events to the Data
+    # Index Service reducing the number of produced events. Set to false to send individual events.
+    kogitoEventsGrouping: true
```

Comment:
Why are we setting this on the controller and not the on platform level? The relationship between the platform and data index is 1:1, so this information should be set there in this case.

Reply:
@ricardozanini I don't want to contaminate the SPF definition with that value, so, in the sonataflowplatform CRD definition we don't add this value, like we did with the dbMigrator parameter.

Instead we have this.

1. At cluster level, people can enable/disable the kogitoEventsGrouping. By default I configured the value true. Since after discussions in our meetings and with @fjtirado it looks like the best suited configuration. So by default, based on the kogitoEventsGrouping, we'll add the corresponding value to the corresponding application.properties.
2. BUT, if people wants to overwrite this value we set by default, for whatever reason, they can go to the corresponding WF, DI, or JS configuraion and set the equivalent env variable, that will overwrite the value the opeator set in the application.properties.

So, with 1 and 2, we have a simple mechanims to configure the kogitoEvensGrouping by default, and, people can always overwrite in particular situations. Which, as it was investigated, is not the recommended approach.

Reply:
It would be okay if users could overwrite this value via ENV or properties. 👍